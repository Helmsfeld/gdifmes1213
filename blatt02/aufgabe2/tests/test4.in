10
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
10000010
20
30
40
50
60
70
80
90
10000
20000
30000
40000
50000
60000
100000