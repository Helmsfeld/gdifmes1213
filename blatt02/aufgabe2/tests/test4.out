0
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
18287481
0
10
39
65
78
97
114
130
18248
36533
54816
73108
91392
109679
182831
